raidar/management/commands/restat.py

```diff
@@ -450,8 +450,8 @@ def calculate_area_stats(era):
 
             area_queryset = Area.objects.all()
             for area in area_queryset.iterator():
-                encounter_queryset = era.encounters.prefetch_related('participations__account').filter(area=area).order_by('?')
-                totals_in_area, leaderboards_in_area = initialise_era_area_stats(encounter_queryset.count())
+                encounter_queryset = era.encounters.filter(area=area).order_by('?')
```

Comment:
Question for anyone who knows. Why isn't this further filtered by encounter.success? I don't see where unsuccessful logs  are used in any way for the analytics...

Reply:
Hmmm I'm not sure tbh. Thinking about it, I think we do filter later on by success..since the amalgamated stats are definitely only based on success logs. 

Failed encounters are still included in personal stats though, so that might be why?

Reply:
Right, but you wouldn't need to fetch them for the era/area computation. Only for the user level.